src/main/java/org/apache/commons/collections4/bag/CollectionSortedBag.java

```diff
@@ -50,7 +52,7 @@ public static <E> SortedBag<E> collectionSortedBag(final SortedBag<E> bag) {
     /**
      * Constructor that wraps (not copies).
      *
-     * @param bag  the sorted bag to decorate, must not be null
+     * @param bag the sorted bag to decorate, must not be null
```

Comment:
Please don't change the formatting in PRs, it makes it harder to review when there is noise compared to actual changes.